### PR TITLE
[INTERNAL] Travis: Activate Windows and Mac builds

### PR DESCRIPTION
**Pull request by RandomByte**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+
+os:
+- linux
+- osx
+- windows
+
+matrix:
+  include:
+    - os: linux
+      node_js:
```
Comment (matz3) [inline .travis.yml]: It seems that this only runs a build with 10, but not with 8, right?

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
+        - "10"
+        - "8"
 
 after_script:
 - npm run report-coveralls
```
Comment (matz3) [inline .travis.yml]: This fails on windows as the npm script tries to set a ENV variable. As it's the `after_script` it doesn't fail the build.

Comment (matz3) [reply .travis.yml]: Also the coverage is different on Linux and Mac. Several branches in `serveIndex.js` differ. Maybe due to different file-systems?
I think for now we could just stick with reporting coverage on linux only. Especially as Coveralls doesn't seem to show much information that would be helpful for comparing differences between environments.

Comment (RandomByte) [reply .travis.yml]: Done

**Pull request merged**